Formula/rpki-client.rb

```diff
@@ -16,9 +17,11 @@ class RpkiClient < Formula
   depends_on "pkg-config" => :build
   depends_on "libressl"
```

Comment:
Also, I'm sure we've had this conversation before, but: why doesn't this use OpenSSL again?

Reply:
> Also, I'm sure we've had this conversation before, but: why doesn't this use OpenSSL again?

Because Job Snijders is in OpenBSD, the code comes from OpenBSD and there are significant issues with the OpenSSL code and ASN.1 handling for certificate extensions, and code complexity. It uses the OpenBSD supported LibreSSL "fork" (I use that word highly inadvisably) to get round these problems. RPKI is a security focussed space for marking Internet Addresses and BGP routing information, Its *all* about security. The OpenBSD people don't want to taint it with OpenSSL which is now impossible to maintain and align to modern code expectations.

Reply:
a local GIT fluent person is helping explain to me with a clue-by-four how to drive this at shell cmd level so I can meet your goals.